public-docs/payments-discounts.md

```diff
@@ -23,6 +23,6 @@ Discount codes can be enabled in the **Payments** <i class="rui font-icon fa fa-
   - sale - overrides item pricing with a fixed sale price
   - shipping - the formula value should be a string that matches the name of a shipping method. The calculated shipping rate will be applied as a discount.
 
-![](/assets/admin-dashboard-payments-discounts.png "Payment Discounts")
+![](/assets/admin-dashboard-payments-discounts-catalyst.png "Payment Discounts")
 
-Discounts are validated when a user enters a code during checkout, and are applied as payment methods on the cart. The discount code usage is tracked once the order has been placed.
+Discounts are validated when a user enters a code during checkout, and are applied as payment methods on the cart. The discount code usage is tracked once the order has been placed.s
```

Comment:
```suggestion
Discounts are validated when a user enters a code during checkout, and are applied as payment methods on the cart. The discount code usage is tracked once the order has been placed.
```

A  <kbd>Command</kbd>+<kbd>s</kbd> in the wild! 😆

Reply:
🤦‍♂ stupid keyboard